lib/jormungandr/exe/cardano-wallet-jormungandr.hs

```diff
@@ -396,7 +399,7 @@ helperTracingText = unlines $
     [ "Additional tracing options:"
     , ""
     , "  --log-level SEVERITY     Global minimum severity for a message to be logged."
-    , "                           Defaults to \"INFO\" unless otherwise configured."
+    , "                           Defaults to \"DEBUG\" unless otherwise configured."
```

Comment:
:+1:

Reply:
Yeah, I had to discard this to make other integration tests happy.